config/locales/devise.zh_CN.yml

```
@@ -1,5 +1,5 @@
 #Additional translations at https://github.com/plataformatec/devise/wiki/I18n
-zh_CN:
+zh-CN:
```

```suggestion
zh_CN:
```

@nithindavid  we need to toggle this back to `zh_CN`